pybossa.js

```diff
@@ -54,13 +54,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     }
 
     function _saveTaskRun(taskrun) {
-        return $.ajax({
-            type: 'POST',
-            url: url + 'api/taskrun',
-            dataType: 'json',
-            contentType: 'application/json',
-            data: taskrun
-        });
+        if(window.pybossa.isGoldMode){
+            const projectId = JSON.parse(taskrun).project_id;
```

Comment:
maybe you can pass the project id from the `_createTaskRun` function

Reply:
do you think is ok to use `const` here?

Reply:
agree